### ISSUE TITLE
Bluetooth Interference in congested environments

### ISSUE THREAD

**Issue by dinhn4** [andengineering/BLE_Sample_Android#15]
How susceptible are these devices to bluetooth interference?

I have it implemented it into an Android app using an Android 4.4 device. It works fine in a home/office environment. After measurement is taken, the Android device picks up the reason correctly.

I then used it at in a "noisier" wireless congested environment. (Ex. Electronics trade show with many WiFi and Bluetooth devices). The reading would never hit my Android device. 

Is this something that is known? Thanks

**Comment by sim0512** [comment]
Yes in an environment where there are multiple BLE devices there could be interferences. 
Can you tell me which A&D device are you using . Is it the 651 BP Monitor or the 352 Weight Scale ?  Since these devices need to be paired and then only when we take a measurement they advertise the interference should be a lot less for these devices
